hoomd/hpmc/pytest/test_quick_compress.py

```diff
@@ -105,6 +110,51 @@ def test_valid_setattr_attached(attr, value, simulation_factory,
     assert getattr(qc, attr) == value
 
 
+@pytest.mark.parametrize("xy", [-0.4, 0, 0.2])
+@pytest.mark.parametrize("xz", [-0.3, 0, 0.3])
+@pytest.mark.parametrize("yz", [-0.2, 0, 0.4])
+@pytest.mark.parametrize("phi", [0.01, 0.4, 0.6])
```

Comment:
```suggestion
@pytest.mark.parametrize("phi", [0.01, 0.4, 0.6])
pytest.mark.cpu
```
I know you got this running on the GPU - but it significantly increases the test run time. As the modified code runs on the CPU, I don't think we learn much by running the test on the GPU.